integration_tests/support/events.ts

```diff
@@ -47,6 +47,7 @@ const sendDomainEvent = async (body: string): Promise<unknown> => {
     MessageBody: body,
   })
   // Wait for the event listener to clear the queue
+  // eslint-disable-next-line no-promise-executor-return
   await new Promise(resolve => setTimeout(resolve, 10000))
```

Comment:
these just need to be wrapped in blocks so they aren't implicit returns: 
```
await new Promise(resolve => { setTimeout(resolve, 10000) } )
```

Reply:
Done